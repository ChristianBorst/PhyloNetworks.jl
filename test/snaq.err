
 Total errors: 0 in seeds []