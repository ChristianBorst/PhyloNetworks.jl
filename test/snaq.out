(6,(7,(10,(8)11#H1:0.25::0.909):0.098):1.108,(4,11#H1:0.088::0.091):0.0);
 -Ploglik = 0.002162891939599794
 Dendroscope: (6,(7,(10,(8)11#H1:0.25):0.098):1.108,(4,11#H1:0.088):0.0);
 Elapsed time: 0.745330161 seconds in 1 successful runs
-------
List of estimated networks for all runs:
 ((7,(10,(8)11#H1:0.081::0.909):0.098):1.108,4,(6,11#H1:0.089::0.091):0.0);, with -loglik 0.0021628920163591636
 (6,(7,(10,(8)11#H1:0.25::0.909):0.098):1.108,(4,11#H1:0.088::0.091):0.0);, with -loglik 0.002162891939599794
-------